README.md

```diff
@@ -532,6 +532,39 @@ ctest
 cmake --install . --prefix <path_to_install_dir>
 ```
 
+### Building for CUDA
+
+- On Linux*
+
+With the cuBLAS backend:
+
+```bash
+# Inside <path to onemkl>
+mkdir build && cd build
+export CXX=<path_to_dpcpp_compiler>/bin/dpcpp;
+cmake .. -DENABLE_CUBLAS_BACKEND=True                      \
+         -DENABLE_MKLCPU_BACKEND=False                     \   # disable Intel MKL CPU backend
+         -DENABLE_MKLGPU_BACKEND=False                     \   # disable Intel MKL GPU backend
+         [-DREF_BLAS_ROOT=<reference_blas_install_prefix>] \   # required only for testing
```

Comment:
Let's have one common style: 
let's specify all options in [], as we have above for oneMKL building

Reply:
@akabalov is this about the formatting using `[]`?
If so, there are some community guidelines that you can find online, about best practices for markdown documentation, as we have here in our `README.md`. All options specified inside `[]` are optional, but all options without them are mandatory for the command to work. `<>` specifies the intended value for a given option. If specifying multiple valid options, one can use `|` to separate them. Something like this: `-DBUILD_DOC=<TRUE|FALSE>`.
Hope that helps! :)

Reply:
I'm not sure what you mean by that, if it's about `ENABLE_MKLGPU_BACKEND` and `ENABLE_MKLCPU_BACKEND`, I specifically made them non-optional because these are enabled by default, so if you simply omit them when trying to build for cuBLAS it will still try to build them as well, and that will fail on a system that doesn't have oneMKL installed.

Obviously on a system with both cuBLAS and oneMKL we could enable them both but for just the CUDA section I feel like it's less confusing to do it this way.

Reply:
@npmiller I think these changes are correct, but let's wait for @akabalov to clarify if this was her original question or something else.